openquakeplatform/openquakeplatform/urls.py

```diff
@@ -84,14 +84,16 @@
         template_name="geodetic.html"), name='geodetic'),
     url(r'^gaf_viewer/$', TemplateView.as_view(
         template_name="gaf_viewer.html"), name='gaf_viewer'),
-    url(r'^grv_viewer/$', TemplateView.as_view(
-        template_name="grv_viewer.html"), name='grv_viewer'),
-    url(r'^irv_viewer/.*$', TemplateView.as_view(
-        template_name="irv_viewer.html"), name='irv_viewer'),
+    url(r'^grv/$', TemplateView.as_view(
+        template_name="grv/grv_viewer.html"), name='grv'),
+    url(r'^irv/.*$', TemplateView.as_view(
+        template_name="irv/irv_viewer.html"), name='irv'),
```

Comment:
The change in this link name should be reflected also in the layer page template, since it is looking for `irv_viewer`

![screenshot from 2015-11-25 10-49-30](https://cloud.githubusercontent.com/assets/1818657/11393847/622d8f5e-9362-11e5-9325-558659efcb4d.png)